Readme.md

```diff
@@ -14,3 +16,10 @@ Some common workflows and actions for Github Actions shared between Lombiq proje
 Bug reports, feature requests, comments, questions, code contributions and love letters are warmly welcome. You can send them to us via GitHub issues and pull requests. Please adhere to our [open-source guidelines](https://lombiq.com/open-source-guidelines) while doing so.
 
 This project is developed by [Lombiq Technologies](https://lombiq.com/). Commercial-grade support is available through Lombiq.
+
+### Code styling
```

Comment:
I still think this is better in our wiki, especially that we actually already have a section for this: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows+and+actions#:~:text=LGHA)%20for%20examples.-,Editor%20support,-%3A These notes are not specific to this project, and on the other hand, the same practices need to be followed by other projects too.

Reply:
I don't disagree with that especially for keeping the same type of information in the same place, but that still means that this information is not visible outside Lombiq. Which might not be an issue now, because we don't have external contributors at the moment (but hopefully we will). On the other hand, YAML linting might alleviate that (but it would still be better, if the code styling rules were clear upfront, not just from e.g., a failed build, but we can worry about that later).